github/actions/multi_client.go

```diff
@@ -142,14 +142,14 @@ func (m *multiClient) GetClientFromSecret(ctx context.Context, githubConfigURL,
 	return m.GetClientFor(ctx, githubConfigURL, auth, namespace, options...)
 }
 
-func RootCAsFromConfigMap(configMapData map[string][]byte) (*x509.CertPool, error) {
+func RootCAsFromConfigMap(configMapData map[string]string) (*x509.CertPool, error) {
 	caCertPool, err := x509.SystemCertPool()
```

Comment:
is there any way we can trace out that the current client is using a proxy or CACert? those would be super helpful during debug

Reply:
We could add these to the client logger when applying options perhaps? Probably best in a separate PR right after this one to save a diff?